src/pages/Stream/Views/Explore/JSONView.tsx

```diff
@@ -61,25 +61,28 @@ const CopyIcon = (props: { log: Log }) => {
 
 const Row = (props: {
 	log: Log;
-	headers: string[];
 	searchValue: string;
 	disableHighlight: boolean;
 	shouldHighlight: (val: number | string | Date | null) => boolean;
 }) => {
-	const { log, headers, disableHighlight, shouldHighlight } = props;
+	const { log, disableHighlight, shouldHighlight } = props;
 
 	return (
 		<Stack style={{ flexDirection: 'row' }} className={classes.rowContainer} gap={0}>
 			<span>
 				{_.isObject(log) ? (
-					_.map(headers, (header, index) => (
-						<Item
-							header={header}
-							key={index}
-							value={_.toString(log[header])}
-							highlight={disableHighlight ? false : shouldHighlight(log[header])}
-						/>
-					))
+					_.map(log, (value, key) => {
+						//skiping fields with empty strings
```

Comment:
Not sure why this is still needed.
If you are iterating over a log object, why you'd want to test whether its empty or not ?